spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { (create(:project) )}
+  let(:task) { create(:task, project: project) }  
```

Comment:
`association`を定義していれば`project`を指定する必要はないと思います。
```suggestion
  let(:task) { create(:task }  
```

Reply:
5a5edf3